addons/im_livechat/static/tests/mock_server/mock_models/discuss_channel_member.js

```diff
@@ -12,7 +12,7 @@ export class DiscussChannelMember extends mailModels.DiscussChannelMember {
         const [member] = this.browse(ids);
         const [channel] = DiscussChannel.browse(member.channel_id);
         if (channel.channel_type === "livechat") {
-            return ["active", "avatar_128", "country", "is_public", "user_livechat_username"];
+            return ["active", "avatar_128", "country_id", "is_public", "user_livechat_username"];
```

Comment:
mock server of res partner should also be updated (I guess that's why most tests are not failing)

Reply:
Even with this change, there is no test failing in livechat or mail.
I guess I'll look at it and add coverage